### PR TITLE
Adding Github workflow to deploy docker image to docker hub 

### PR DESCRIPTION
**Pull request by abdorah**
I tried to add a Github workflow to deploy a docker image to the docker hub. _(first sorry I forget the end of the line so the pre-commit check failed)_. This workflow needs two Github secrets to work: `secrets.DOCKER_HUB_USERNAME`, and `secrets.DOCKER_HUB_PASSWORD`

### CODE REVIEW

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        uses: docker/login-action@v1
+        with:
+          username: ${{secrets.DOCKER_HUB_USERNAME}}
+          password: ${{secrets.DOCKER_HUB_PASSWORD}}
+      - name: Build and Push to DockerHub
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          tags: ${{secrets.DOCKER_HUB_USERNAME}}/one:v1
```
Comment (jbampton) [inline .github/workflows/docker-image.yml:32]:
```suggestion
          tags: ${{secrets.DOCKER_HUB_USERNAME}}/one:v1
```

File: .github/workflows/docker-image.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+name: Dockerize One Programming language
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+
```
Comment (jbampton) [inline .github/workflows/docker-image.yml:10]:
```suggestion
```

File: .github/workflows/docker-image.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  build-deploy:
+    name: Build and Publish Docker image
+    runs-on: ubuntu-latest
+
```
Comment (jbampton) [inline .github/workflows/docker-image.yml:14]:
```suggestion
```

**Review by abdorah — COMMENTED** [review]
Hey, @jbampton I updated the code but I don't think there is any need to open a new PR, but please don't accept this PR till this [job](https://github.com/abdorah/One/runs/3203410594) ends.
Thank you for your reviews.

**Comment by abdorah** [comment]
Hey @jbampton and @BaseMax ! You can check now the PR now. This is [docker hub repository for one ](https://hub.docker.com/r/onelangorg/one)

**Comment by BaseMax** [comment]
**DOCKER_HUB_USERNAME**, and **DOCKER_HUB_PASSWORD** secrets added.

**Pull request merged**